frontend/src/main/dc-migration-assistant-fe/api/migration.ts

```
@@ -98,4 +99,20 @@ export const migration = {
             res.json()
         );
     },
+    resetMigration: (): Promise<void> => {
```

This is a dev endpoint right? Should we throw an error if this is called in production?

---

It will 404, as the endpoint won't exist if the profile isn't enabled.

---

After this is https://github.com/atlassian-labs/dc-migration-assistant/pull/169 merged, it is production endpoint but it requires the `error` stage.